PharmaProject.Server/Program.cs

```diff
@@ -18,11 +18,15 @@
 string connectionString = builder.Configuration.GetConnectionString("AppSettingsDbContext");
 
 builder.Services.AddDbContext<AppSettingsDbContext>(options =>
```

Comment:
check out [AddDbContextPool](https://learn.microsoft.com/en-us/dotnet/api/microsoft.extensions.dependencyinjection.entityframeworkservicecollectionextensions.adddbcontextpool?view=efcore-8.0). It's an easy thing to implement that'll improve the APIs scalability